### ISSUE TITLE
indentSwitchCase not compatible with final release of ESLint v1.0.0

### ISSUE THREAD

**Issue by NassarS** [feross/eslint-config-standard#6]
I get an error running eslint . --ext .js with 

"eslint": "^1.0.0",
"eslint-config-standard": "^4.0.0-5",

Error: eslint-config-standard:
	Configuration for rule "indent" is invalid:
	Value "data["2"].indentSwitchCase" has additional properties.

It appears that shortly before the 1.0.0 launch, eslint was changed to use the rule

```
"indent": [2, 2, { "SwitchCase": 1 }],
```

instead of:

```
"indent": [2, 2, { "indentSwitchCase": true }],
```

See:
https://github.com/eslint/eslint/commit/333fb6c4ebb2d3359650c52a11b1f3ce04ae45cc#diff-c391782b252c49c8bc25c8769afa7eb7

**Comment by feross** [comment]
This is fixed! 4.0.0 is released.

**Comment by ghostbar** [comment]
For anyone coming here with this issue: upgrade your eslint.
